cmd/litestream/main_windows.go

```diff
@@ -1,11 +1,10 @@
-// +build windows
+//go:build windows
 
 package main
 
 import (
 	"context"
 	"io"
-	"log"
```

Comment:
"log" package is still used; see lines 38-41,49.

Reply:
Right, the Windows build isn't built anymore but the code still exists. It's basically unmaintained so I didn't notice it since I didn't do a Windows build.

I guess it could be fixed but it doesn't really change much.

Reply:
For what it's worth, Litestream works just fine on Windows after adding imports.